src/components/start/start.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {StyleSheet, Text, View} from 'react-native';
+
+import Button from "../shared/Button/Button";
+
+import {AntDesign} from '@expo/vector-icons';
+import MaterialCommunityIcons from "react-native-vector-icons/MaterialCommunityIcons";
+export default function Start({setScreen}) {
+    return (
+        <View style={styles.container}>
+            <View>
+                <Text style={styles.text}>Dark side of <Text style={styles.innerText}>Book store</Text></Text>
+                <View style={styles.buttonContainer}>
+                    <Button text='Login' onPress={() => setScreen('login')}/>
+                    <Button text='SignUp' type='dark' onPress={() => setScreen('signUp')}/>
+                </View>
+                <Text style={styles.textMedia}>Social <Text style={styles.innerText}>media</Text></Text>
+            </View>
+            <View style={styles.iconContainer}>
+                <AntDesign name="facebook-square" style={styles.icon}/>
```

Comment:
I could be wrong, but you are trying to use two different types of icons in your projects: Feather and AntDesign. Do you have really strong reason to be inconsistent with icons styles?